shared/src/common/types/user/user-sign-up-response-dto.type.ts

```diff
@@ -3,4 +3,9 @@ type UserSignUpResponseDto = {
   email: string;
 };
 
-export { type UserSignUpResponseDto };
+type UserSignUpTokenResponseDto = {
+  token: string;
+  user: UserSignUpResponseDto;
+};
```

Comment:
```suggestion
type UserSignUpResponseDto = {
  token: string;
  user: UserSignUpResponseDto;
};
```
we should use one type for it

Reply:
but what if in actions I dont need to return token, only user?

Reply:
Create a new type `UserByIdResonse` and use in two places

```
type UserSignUpResponseDto = {
  token: string;
  user: UserByIdResonse;
};
```
and in the action